catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/PaimonCatalogOperations.java

```diff
@@ -21,6 +21,8 @@
 import static org.apache.gravitino.catalog.lakehouse.paimon.GravitinoPaimonTable.fromPaimonTable;
 import static org.apache.gravitino.catalog.lakehouse.paimon.PaimonSchema.fromPaimonProperties;
 import static org.apache.gravitino.connector.BaseCatalog.CATALOG_BYPASS_PREFIX;
+import static org.apache.gravitino.rel.expressions.transforms.Transforms.EMPTY_TRANSFORM;
+import static org.apache.paimon.utils.Preconditions.checkArgument;
```

Comment:
should use `com.google.common.base.Preconditions.checkArgument`, instead of 
 `org.apache.paimon.utils.Preconditions.checkArgument`

Reply:
updated.